_RFCs/0004-testing-pull-requests.md

```diff
@@ -0,0 +1,48 @@
+- Change Name: testing_requirements_pull_requests
+- Start Date: 2018-03-27
+- RFC PR: (leave this empty)
+- Core-Plans Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Pull requests to core plans (whether new core plans or updates to existing ones) must have steps for manually testing the change (which will be automated as much as possible when the automated testing framework is available).
+
+# Motivation
+[motivation]: #motivation
+
+Stability and reliability of core plans is central to the user experience and is therefore a top priority of core plans maintainers. Having a way to test an updated or new core plan helps improve the quality of core plans, and provides confidence in the change submitted.
+
+Part of what makes pull requests to core plans linger open for so long is that, although many people are comfortable with reviewing the code, many are hesitant to merge, release, and promote the package for fear of breaking it or other packages dependent on it.  A future automated testing framework (currently in development) will automate much of this testing, but before we can automate testing something, we need to know how to test it manually.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+When someone opens a pull request to the core plans repo, they must include steps to test the change they are making (or the functionality of a new plan). The intent is to have a way to test any new functionality and, as much as possible, prevent breaking of existing functionality. Providing these testing steps (and sample apps when useful) is the responsibility of the person opening the pull request. However, should the person opening the pull request need help with establishing testing steps, they are most welcome to ask for it in the pull request.  A pull request must not be approved until testing steps are provided in the pull request.
```

Comment:
How is this communicated to the user?
This could also use language around leveraging an existing test suite for the plan.

Reply:
Good question.  Should I see a PR without steps for testing the plan, for now I will ask them in the PR.  This will become more automated in the future - especially once @eeyun and my integration test work is completed.